scripts/docker/deploy/entrypoint.sh

```diff
@@ -73,7 +75,7 @@ if [ "$1" = 'arangod' ]; then
 
         if [ ! -z "${ARANGO_ROOT_PASSWORD+x}" ]; then
             echo "Initializing root user...Hang on..."
-            ARANGODB_DEFAULT_ROOT_PASSWORD="$ARANGO_ROOT_PASSWORD" /usr/sbin/arango-init-database -c /tmp/arangod.conf --server.rest-server false --log.level error --database.init-database true || true
+            ARANGODB_DEFAULT_ROOT_PASSWORD="$ARANGO_ROOT_PASSWORD" /usr/sbin/arango-init-database -c /tmp/arangod.conf $@ --server.rest-server false --log.level error --database.init-database true || true
```

Comment:
It looks inconsistent to me that we use `$@` here without putting it in quotes, but in the two other places at which the PR introduces the usage of `$@`, it is put in quotes.